fixtures/lang-frontends/python-flask-fastapi/computed.py

```
@@ -0,0 +1,6 @@
+prefix = "/api"
+
+
+@app.route(prefix + "/users")
```

Ruff (0.16.1)

[error] Undefined name `app`

(F821)